CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.29.1] - 2025-02-18
```

Comment:
need to add 0.29.1 also at the very bottom to the list of links

Reply:
Done, though it'll be a broken link until the tag is pushed.